src/extern/bpf/bpf.h

```diff
@@ -0,0 +1,218 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
```

Comment:
this heder file already exists as lib/usr/include/bpf/bpf.h.

Reply:
out of box bpf.h does not have batch update:
https://man7.org/linux/man-pages/man2/bpf.2.html
That's why I have to manually add it.